DeadCrumbs/app/src/main/java/dead/crumbs/ui/MainActivity.kt

```diff
@@ -136,7 +139,8 @@ class MainActivity : AppCompatActivity() {
             // We've bound to BluetoothService, cast the IBinder and get LocalService instance
             val binder = service as BluetoothService.LocalBinder
             mService = binder.getService()
-            mService.callback = fun(rssi:BluetoothRSSI) { viewModel!!.addRSSI(rssi) }
+            mService.callback = fun(rssi:BluetoothRSSI)
```

Comment:
Just personal shit, but I thought line 142 and 143 where to two different statements, so maybe move the curly bracket on line 143 up on 142. Also, is this the normal way of declaring a lambda? Or is it not a lambda? Normally we have something like { params -> body }